src/components/SearchBox.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const SearchBox = () => {
+  return (
+    <form>
+      <input
+        type="text"
+        placeholder="Search for artists, bands, tracks, podcasts"
+      />
+      <button type="submit">
+        <i class="fa fa-search"></i>
```

Comment:
Please could you use the svg asset attached to the story, rather than font awesome